src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala

```diff
@@ -36,6 +36,67 @@ trait ResourceRoutes extends UserInfoDirectives with SecurityDirectives with Sam
       case None => throw new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.NotFound, s"resource type ${name.value} not found"))
     }
 
+  def adminResourceRotues: server.Route =
```

Comment:
TOL: I had actually been thinking this should maybe be in a new file, since the routes aren't necessarily related (although there is a lot of shared functionality)

Reply:
Good call, I followed the admin routes pattern we used for admin user routes, but I think it's separate enough to pull into a different file